postgresql-containers/build/postgres-postgis/Dockerfile

```diff
@@ -0,0 +1,216 @@
+FROM golang:1.20 AS go_builder
+WORKDIR /go/src/github.com/mikefarah/yq
+
+ARG GOOS=linux
+ARG GOARCH=amd64
+ARG CGO_ENABLED=0
+
+ENV YQ_VERSION 3.3.4
+RUN git clone -b $YQ_VERSION https://github.com/mikefarah/yq.git .; \
+    go get -u ./...; \
+    mkdir -p build/_output/bin; \
+    ./scripts/devtools.sh; \
+    sed -i -e 's^dev: test ^dev: ^' ./Makefile; \
+    CGO_ENABLED=$CGO_ENABLED GOOS=$GOOS GOARCH=$GOARCH \
+    make local build
+
+FROM redhat/ubi8-minimal
```

Comment:
I think we have use our postgres image as a base image

Reply:
Check it out @hors pls: [6e1c2a4](https://github.com/percona/percona-docker/pull/857/commits/6e1c2a43371a99b887d90357404b8247b73ce188)